plottable.d.ts

```diff
@@ -1502,6 +1503,24 @@ declare module Plottable {
          */
         anchor(selection: D3.Selection): Component;
         /**
+         * Adds a callback to be called on anchoring the Component to the DOM.
+         * If the component is already anchored, the callback is called immediately.
+         *
+         * @param {AnchorCallback} callback The callback to be called on Anchor
+         *
+         * @return {Component}
+         */
+        onAnchor(callback: AnchorCallback): Component;
+        /**
+         * Removes a callback to be called on anchoring the Component to the DOM.
+         * The callback is identified by reference equality.
+         *
+         * @param {AnchorCallback} callback The callback to be removed
```

Comment:
I liked the `callback to be added / removed` idea. Changing to that to have better symmetry